doc/package-android.md

```diff
@@ -0,0 +1,12 @@
+Android package guide
```

Comment:
Should this file be linked to from the doc/Readme?

Also, should it be named build-android, similar to build-unix and build-bsd?

Reply:
I thought so but other build files seem to be about building _on_ UNIX, BSD, etc... I'm open to suggestions.

Reply:
Oh, this is for cross-compilation? Well, we have build-win

Reply:
You are right. I will change to build-android and make it a bit more generic (as in not just about packaging)